patients/get_appointments.php

```diff
@@ -1,58 +1,100 @@
 <?php
- // Copyright (C) 2011 Cassian LUP <cassi.lup@gmail.com>
- //
- // This program is free software; you can redistribute it and/or
- // modify it under the terms of the GNU General Public License
- // as published by the Free Software Foundation; either version 2
- // of the License, or (at your option) any later version.
- //
- // Modified to support recurring appointments by Ian Jardine 2016.
+/**
+ *
+ * Copyright (C) 2016-2017 Jerry Padgett <sjpadgett@gmail.com>
+ * Copyright (C) 2011 Cassian LUP <cassi.lup@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author Cassian LUP <cassi.lup@gmail.com>
+ * @author Jerry Padgett <sjpadgett@gmail.com>
+ * @link http://www.open-emr.org 
+ *
+ */
+ require_once ("verify_session.php");
 
-	require_once("verify_session.php");
-	///////
-	require_once(dirname(__FILE__)."/../library/appointments.inc.php");
+$query = "SELECT e.pc_eid, e.pc_aid, e.pc_title, e.pc_eventDate, " . "e.pc_startTime, e.pc_hometext, e.pc_apptstatus, u.fname, u.lname, u.mname, " . 
```

Comment:
Should be able to revert this code block to the original code (which uses fetchNextXAppts) and it should work. I think what happened here is that the fetchNextXAppts stuff and related code(in order to support recurrent appt) here were added after you branched your code to work on the portal.